<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimensionPrefix_Dialog</class>
 <widget class="QDialog" name="DimensionPrefix_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dimension tolerance text options</string>
  </property>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text font</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>36</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>62</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>334</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>334</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>88</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>38</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Horiz just</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>88</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert spacing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>62</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert just</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="_104">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>36</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="_105">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>112</y>
     <width>317</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Style</string>
   </property>
   <widget class="QCheckBox" name="_107">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>21</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_109">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>43</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_110">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>42</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Boxed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_108">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>22</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
  </widget>
  <widget class="RCDoubleSpinBox" name="_101">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>34</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>60</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>86</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_106">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>86</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_111">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>192</y>
     <width>319</width>
     <height>139</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>319.000000000000000</width>
     <height>139.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="transformationAnchor">
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::FullViewportUpdate</enum>
   </property>
  </widget>
  <widget class="QFontComboBox" name="_100">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>6</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentFont">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="_1000">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>1</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="_1004">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>31</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="_1005">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>65</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <zorder>_1005</zorder>
  <zorder>_1004</zorder>
  <zorder>_1000</zorder>
  <zorder>label_45</zorder>
  <zorder>label_53</zorder>
  <zorder>label_54</zorder>
  <zorder>_2</zorder>
  <zorder>_1</zorder>
  <zorder>label_56</zorder>
  <zorder>label_57</zorder>
  <zorder>label_58</zorder>
  <zorder>label_59</zorder>
  <zorder>_104</zorder>
  <zorder>_105</zorder>
  <zorder>groupBox</zorder>
  <zorder>_101</zorder>
  <zorder>_102</zorder>
  <zorder>_103</zorder>
  <zorder>_106</zorder>
  <zorder>_111</zorder>
  <zorder>_100</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
